ui/storybook/package.json

```diff
@@ -35,9 +35,15 @@
     "babel-plugin-react-require": "^3.1.3",
     "happo-plugin-storybook": "^3.2.0",
     "happo.io": "^8.3.1",
+    "msw": "^1.2.1",
+    "msw-storybook-addon": "^1.8.0",
```

Comment:
This is not an internal storybook addon, but it is the most popular addon for mocking data fetching and seems to be kept up-to-date.